ts/doc.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Matt Tracy (matt.r.tracy@gmail.com)
+
+/*
+Package ts provides a high-level time series API on top of an underlying
+cockroach datastore. Cockroach itself provides a single monolithic, sorted
+key-value map. The ts package collects time series data, consisting of
+timestamped data points, and efficiently stores it in the key-value store.
+
+Storing time series data is a unique challenge for databases. Time series data
+is typically generated at an extremely high volume, and is queried by providing
+a range of time of arbitrary size, which can lead to an enormous amount of data
+being scanned for a query. Many specialized time series databases exist to meet
+these challenges.
+
+Cockroach organizes time series data by series name and timestamp; when querying
+a given series over a time range, all data in that range is thus sorted
+contiguously. However, cockroach transforms the data in a few important ways in
+order to effectively query a large amount of data.
+
+###Downsampling
```

Comment:
Should there be a space between "###" and "Downsampling"? Not sure how this will look in godoc.